.github/workflows/stylish.yml

```diff
@@ -0,0 +1,38 @@
+name: stylish
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  stylish:
+    name: "black & flake8 & rpmlint"
+    runs-on: ubuntu-latest
+    container:
+      image: fedora:latest
+
+    steps:
+    - name: Base setup
+      run: |
+        dnf --setopt install_weak_deps=False install -y \
+            git-core \
+            python3-flake8 \
```

Comment:
I don't understand. Do we define version of flake8 anywhere? I cannot see it.

Reply:
We have flake8 in `test-requirements.txt` (and also in `setup.py`, as of course our build system is not easy enough...).
Until some time ago it was pinned to a version earlier than 4, as flake8 4.x dropped support for `distutils`; now we simply run `flake8(1)`, so we aren't limited anymore.

I don't have anything against installing it from pip, however I'm slightly worried that it might break due to brand-new-shining-and-breaking version, whereas using the distro version of the latest Fedora (currently 36) is a bit more stable (as in, changing less often).